.github/workflows/publish-maven-snapshots.yml

```
@@ -0,0 +1,38 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+        - main
+        - 2.x
```

```suggestion
        - 1.x
```

We don't have 2.x (yet) for this project